provisioning_templates/PXEGrub/pxegrub_global_default.erb

```diff
@@ -3,8 +3,12 @@ kind: PXEGrub
 name: PXEGrub global default
 %>
 <%# Used to boot unknown hosts, do not associate or change the name. %>
-
-default=0
+<%
+  # Grub1 only supports numeric default statement, this array allows conversion to number
+  items = ["local", "discovery"]
+  default_item = items.index(global_setting("default_pxe_item_global", "local"))
```

Comment:
What happens, if the item is not found? Shouldn't we raise some error with explanation what went wrong?

Reply:
I think a sane approach is to add `|| 0` which works with both PXELinux and Grub. Let me add that and rebase.